src/MobileUI/ViewModels/ProfileViewModels/OtherProfileViewModel.cs

```diff
@@ -8,14 +8,13 @@ public OtherProfileViewModel(IRewardService rewardsService, IUserService userSer
     {
     }
 
-    public override async Task Initialise()
+    public async Task Initialise()
     {
         if (DeviceInfo.Platform == DevicePlatform.iOS)
         {
             MainThread.BeginInvokeOnMainThread(() =>
             {
                 ProfileSections = new ObservableCollection<ProfileCarouselViewModel>();
-                OnPropertyChanged(nameof(ProfileSections));
```

Comment:
This is here as a workaround for a bug and will still be required. Although will be irrelevant in v3 so probably not a big deal.

Reply:
Done - reverted.